scripts/devops_tasks/setup_execute_tests.py

```diff
@@ -301,21 +301,23 @@ def execute_global_install_and_test(
     else:
         target_dir = root_dir
 
-    targeted_packages = discover_targeted_packages(args.glob_string, target_dir, "", args.filter_type)
-    compatible_targeted_packages = filter_packages_by_compatibility_override(targeted_packages)
```

Comment:
This seems to be the only use of `filter_packages_by_compatibility_override` -- should we still hold onto that method?

Reply:
Check within `discover_targeted_packages` 

https://github.com/Azure/azure-sdk-for-python/blob/fc9e6f937141977b326eaffd5bd168096cee40cd/tools/azure-sdk-tools/ci_tools/functions.py#L112-L145

Reply:
That's using `filter_for_compatibility` though; `filter_packages_by_compatibility_override` seems to be doing something slightly different, but at a glance I'm not sure if it's a duplicate or slightly different in functionality

Reply:
I will clean up filter_for_compatibility and filter_packages_by_compatibility_override into a single function. Good catch.

Reply:
Done! Thanks for the follow-up on this one McCoy!